ui-grid/ui-grid-tests.ts

```diff
@@ -41,6 +41,9 @@ columnDef.filter = {
     selectOptions: [{value: 4, label: 'test'}],
     disableCancelButton: false
 };
+columnDef.filter.condition = (searchTerm:string, cellValue:any, row:uiGrid.IGridRow, column:uiGrid.IGridColumn):boolean => {
```

Comment:
The rest of the file has a space between the : and the type.  Please make consistent...

```
columnDef.filter.condition = (searchTerm: string, cellValue: any, row: uiGrid.IGridRow, column: uiGrid.IGridColumn): boolean => {
```